src/main/java/bio/terra/cli/businessobject/Resource.java

```diff
@@ -88,7 +89,8 @@ protected Resource(ResourceMetadata metadata) {
 
   /**
    * Deserialize to the internal representation of the resource from the WSM client library format.
-   * Calls the appropriate sub-class constructor based on the resource type.
+   * Calls the appropriate sub-class constructor based on the resource type. Returns null if the
+   * resource is unsupported by terra-cli.
    */
   public static Resource deserializeFromWsm(ResourceDescription wsmObject) {
```

Comment:
Yeah -- I changed this to avoid failing when there are unknown resources, but I agree that a combination of a warning + providing some information about the unknown resources seems better.